opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -2129,19 +2131,24 @@ namespace Opm {
             changed = true;
             updateAndCommunicate(reportStepIdx, iterationIdx, deferred_logger);
         }
+
+        //if (iterationIdx > 1) {
```

Comment:
This (`// if (...)`) looks like leftover debugging code.  If it is, then please remove it.  If not, please add a comment for why it is or will be needed.